### PR TITLE
Handle unexisting/deleted pages

### PR DESCRIPTION
**Pull request by ValentinBrclz**
For now, since "data" is undefined when the requested page is not existing, a type error occurs. This commit just handle the error.

### CODE REVIEW

File: lib/bot.js [hunk 1]
```diff
@@ -372,13 +372,16 @@ module.exports = (function() {
 					return;
 				}
 
+				if(data === null)
+					callback(title+" does not exist", null);
```
Comment (macbre) [inline lib/bot.js]: Please create an instance of `Error` and then pass it to the callback (nodejs convention).

File: lib/bot.js [hunk 2]
```diff
@@ -372,13 +372,16 @@ module.exports = (function() {
...
+
```
Comment (macbre) [inline lib/bot.js]: I guess we're missing `return` here.

**Comment by ValentinBrclz** [comment]
Here you go. Sorry for that, still learning here =)

**Comment by macbre** [comment]
No worries :) Thanks for the fix!

**Pull request merged**